pwa-shell/src/state/thunks.js

```diff
@@ -262,7 +297,10 @@ export const login = (data) => async dispatch => {
     const json = await response.json();
     dispatch(loginUser(data.username, json.access_token));
   } catch (err) {
-    const msg = err.message === 'permissionDenied' ? 'Username e password non validi' : 'Errore durante il login';
+    const msg =
+      err.message === 'permissionDenied'
```

Comment:
`<FormattedMessage>` missing here

Reply:
I can't directly use formattedmessage in thunks, and after looking for solutions on using the said method, the only solution (and suggestion to devs) as I search around is to pass `intl` object from `injectIntl`, then passing it to actions/thunks. It maybe not a fancy solution but this is probably the only way to integrate react-intl into thunk functions. Let me know what you think.

Reply:
I think the best way should be avoiding i18n outside components, e.g. in thunks.
In order to fix this code, we can:
- make `thunks.js` handle only message ids
- translate messages in `ErrorsAlertContainer`

Reply:
In the end, I created a custom alert container for login, given that the context is slightly different.